unit_tests/test_coredump.py

```diff
@@ -22,16 +22,17 @@ def wait_ssh_up(self, verbose=False):
 class CoredumpExportSystemdTestThread(CoredumpExportSystemdThread):
     lookup_period = 0
 
-    def __init__(self, node: 'BaseNode', max_core_upload_limit: int):
+    def __init__(self, node: 'BaseNode', max_core_upload_limit: int, max_coredump_thread_exceptions: int):
         self.got_cores = []
-        super().__init__(node, max_core_upload_limit)
+        self.raised_exceptions = []
+        super().__init__(node, max_core_upload_limit, max_coredump_thread_exceptions)
```

Comment:
See comments above about class variable and `raised_exceptions` -> `exception`.

Reply:
changed